Fix : code style for product-details & cart components

Piyush132000 opened the pull request:
I have removed unnecessary code from the components & i initialize the form in OnInit function (life cycle). I think initilization of forms and getting data from service must be done inside  ngOnInit.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

I updated the code style & formating the local variables. I moved forms initialization in ngOnInit & also the get items from service. I also removed unnecessary imports.

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Same behavior as previous


Issue Number: N/A


## What is the new behavior?
No any new behavior 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

Piyush132000 committed:
docs: fix code style

I have removed unnecessary code from the components & i initialize the form in OnInit function (life cycle). I think initilization of forms and getting data from service must be done inside OnngOnInit.

jessicajaniuk commented:
@Piyush132000 Looks like this needs some cleanup work to fix broken tests before we can review / land it.

Piyush132000 commented:
> @Piyush132000 Looks like this needs some cleanup work to fix broken tests before we can review / land it.

Sorry for inconvenience, can you please help me how can I remove these failure?

jessicajaniuk commented:
@Piyush132000 You should be able to click in to the failing tests and see what's wrong. There's a details link for each failure test suite that takes you to the circleci test run. There's some very simple linting issues in a few of the suites that should be easy fixes. The test_docs_examples tests have some more legitimate failures though.

Piyush132000 commented:
> @Piyush132000 You should be able to click in to the failing tests and see what's wrong. There's a details link for each failure test suite that takes you to the circleci test run. There's some very simple linting issues in a few of the suites that should be easy fixes. The test_docs_examples tests have some more legitimate failures though.

Sorry for the inconvenience, @jessicajaniuk I tried but was not able to fix the failed test.

Piyush132000 commented:
> > @Piyush132000 You should be able to click in to the failing tests and see what's wrong. There's a details link for each failure test suite that takes you to the circleci test run. There's some very simple linting issues in a few of the suites that should be easy fixes. The test_docs_examples tests have some more legitimate failures though.
> 
> Sorry for the inconvenience, @jessicajaniuk I tried but was not able to fix the failed test.

Hii @jessicajaniuk  can you please help me more so I can fix these failures.

jessicajaniuk commented:
I'm actually not seeing anything here that should be causing these tests to fail. I'm going to rebase for you and see if that resolves the issue.

jessicajaniuk commented on aio/content/examples/getting-started/src/app/cart/cart.component.ts:
```diff
@@ -29,6 +28,19 @@ export class CartComponent {
+    this.items = this.cartService.getItems();
+  }
+
+// #enddocregion initialize form, checkout-form group
```
This is actually the issue. The region parser has been failing, and it's because these regions don't exist. You can cut this line. `checkout-form-group` was closed on line 29 and `initialize form` doesn't appear to exist. Once you delete this line, things should be green.

gkalpak reviewed (changes requested):
Thx for the fixes, @Piyush132000 :+1:
Just doing a drive-by review :smiley:

Piyush132000 committed:
docs: fix code style

I have removed unnecessary code from the components & i initialize the form in OnInit function (life cycle). I think initilization of forms and getting data from service must be done inside ngOnInit.

Piyush132000 committed:
docs: fix code style

Piyush132000 committed:
docs: fix code style

dylhunn commented:
Closing this as there are no recent updates, and it is not mergeable in the current state due to the test failures. Feel free to reopen or @ me if you still intend to work on it.

Piyush132000 commented:
Hii @dylhunn, sorry I forget about this PR. Please reopen it.


Thanks
